File: Solutions/Centauros.App.Web.Mvc/Content/js/app/model/Athlete.js
Ext.define('Centauros.model.Athlete', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'id', type: 'int', mapping: 'Id' },
        { name: 'name', type: 'string', mapping: 'Name' },
        { name: 'birthDate', type: 'date', mapping: 'BirthDate' },
        { name: 'gender', type: 'string', mapping: 'Gender' },
        { name: 'weight', type: 'auto', mapping: 'Weight' },
        { name: 'height', type: 'auto', mapping: 'Height' },
        { name: 'firstTrainingDate', type: 'date', mapping: 'FirstTrainingDate' },
        { name: 'position', type: 'int', mapping: 'Position' },
        { name: 'experience', type: 'int', mapping: 'Experience' },
        { name: 'telephone', type: 'string', mapping: 'Telephone' },
        { name: 'cellphone', type: 'string', mapping: 'Cellphone' },
        { name: 'email', type: 'string', mapping: 'Email' },
        { name: 'urgencyContactName', type: 'string', mapping: 'UrgencyContactName' },
        { name: 'urgencyContactTelephone', type: 'string', mapping: 'UrgencyContactTelephone' },
        { name: 'address', type: 'string', mapping: 'Address' },
        { name: 'addressComplement', type: 'string', mapping: 'AddressComplement' },
        { name: 'cityId', type: 'int', mapping: 'CityId' },
        { name: 'neighborhoodId', type: 'int', mapping: 'NeighborhoodId' },
        { name: 'healthCarePlanId', type: 'int', mapping: 'HealthCarePlanId' },
        { name: 'bloodType', type: 'string', mapping: 'BloodType' },
        { name: 'injuriesHistory', type: 'string', mapping: 'InjuriesHistory' },
        { name: 'allergiesDiseases', type: 'string', mapping: 'AllergiesDiseases' },
        { name: 'cpf', type: 'string', mapping: 'Cpf' },
        { name: 'cpfPath', type: 'string', mapping: 'CpfPath' },
        { name: 'rg', type: 'string', mapping: 'Rg' },
        { name: 'rgPath', type: 'string', mapping: 'RgPath' },
        { name: 'fgrCardNumber', type: 'string', mapping: 'FgrCardNumber' },
        { name: 'fgrCardPath', type: 'string', mapping: 'FgrCardPath' },
        { name: 'irbLawsPath', type: 'string', mapping: 'IrbLawsPath' },
        { name: 'irbReadyPath', type: 'string', mapping: 'IrbReadyPath' },
        { name: 'medicalAttestPath', type: 'string', mapping: 'MedicalAttestPath' },
        { name: 'healthCarePlanCardPath', type: 'string', mapping: 'HealthCarePlanCardPath' },
        { name: 'healthCarePlanCardNumber', type: 'string', mapping: 'HealthCarePlanCardNumber' },
        { name: 'createdOn', type: 'date', mapping: 'CreatedOn' }
	],
    validations: [
        {type: 'presence',  field: 'name'}
    ],
    idProperty: 'id',
    proxy: {
        type: 'rest',
        url: 'Athlete/Create'
    }
});

File: Solutions/Centauros.App.Domain/Athlete.cs
namespace Centauros.App.Domain
{
    using SharpArch.Domain.DomainModel;
    using System;
    using System.ComponentModel.DataAnnotations;

    public class Athlete : Entity
    {
        public Athlete()
        {
            CreatedOn = DateTime.Now;
        }

        [Required]
        public virtual string Name { get; set; }
        [Required]
        public virtual DateTime BirthDate { get; set; }
        [Required]
        public virtual char Gender { get; set; }
        [Required]
        public virtual float Weight { get; set; }
        [Required]
        public virtual float Height { get; set; }
        [Required]
        public virtual DateTime FirstTrainingDate { get; set; }
        [Required]
        public virtual int Position { get; set; }
        [Required]
        public virtual int Experience { get; set; }
        public virtual string Telephone { get; set; }
        [Required]
        public virtual string Cellphone { get; set; }
        [Required]
        public virtual string Email { get; set; }
        [Required]
        public virtual string UrgencyContactName { get; set; }
        [Required]
        public virtual string UrgencyContactTelephone { get; set; }
        [Required]
        public virtual string Address { get; set; }
        public virtual string AddressComplement { get; set; }
        [Required]
        public virtual City City { get; set; }
        [Required]
        public virtual Neighborhood Neighborhood { get; set; }
        [Required]
        public virtual HealthCarePlan HealthCarePlan { get; set; }
        [Required]
        public virtual string BloodType { get; set; }
        public virtual string InjuriesHistory { get; set; }
        public virtual string AllergiesDiseases { get; set; }
        [Required]
        public virtual string Cpf { get; set; }
        public virtual string CpfPath { get; set; }
        [Required]
        public virtual string Rg { get; set; }
        public virtual string RgPath { get; set; }
        public virtual string FgrCardNumber { get; set; }
        public virtual string FgrCardPath { get; set; }
        public virtual string IrbLawsPath { get; set; }
        public virtual string IrbReadyPath { get; set; }
        public virtual string MedicalAttestPath { get; set; }
        public virtual string HealthCarePlanCardPath { get; set; }
        public virtual string HealthCarePlanCardNumber { get; set; }
        public virtual DateTime CreatedOn { get; set; }
    }
}